app/services/katello/ui_notifications/pulp/proxy_disk_space.rb

```diff
@@ -0,0 +1,48 @@
+module Katello
+  module UINotifications
+    module Pulp
+      class ProxyDiskSpace
+        class << self
+          def deliver!
+            SmartProxy.unscoped.with_content.each do |proxy|
+              percentage = proxy.statuses[:pulp].storage['pulp_dir']['percent']
+              if percentage[0..2].to_i < 90 && notification_already_exists?(subject)
+                blueprint.notifications.where(subject: subject).destroy_all
```

Comment:
Getting an error on line 9 and 10 because 'subject' is not defined, i think you meant 'proxy' in both cases